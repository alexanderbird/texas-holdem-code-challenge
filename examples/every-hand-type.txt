2C 4S KS
Lynde 4C 6C 5C 9C
Annalee 2S 3S 5S 6S
Erminie 4D 4H 7D 7C
Brittany JC TH 8C AS
Devon 9D 3H 4D AS
Rachel 8C 8H TC TS
Trixy 3D 5D AC 8H
Krissy AS QS JS TS
Puff 3C 3H 3D 5C
Damita KC KH KD 2D

---

1 Krissy Royal flush
2 Annalee Straight flush 6
3 Damita Four of a kind King
4 Erminie Full house 4 7
5 Lynde Flush 9
6 Trixy Straight 5
7 Puff Three of a kind 3
8 Rachel Two Pair 10 8
9 Devon Pair 4
10 Brittany Highcard Ace
