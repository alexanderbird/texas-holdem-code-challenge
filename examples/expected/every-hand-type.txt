1 Krissy Royal flush
2 Annalee Straight flush 6
3 Damita Four of a kind King
4 Erminie Full house 4 7
5 Lynde Flush 9
6 Trixy Straight 5
7 Puff Three of a kind 3
8 Rachel Two Pair 10 8
9 Devon Pair 4
10 Brittany Highcard Ace
